target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wax50re.c

```diff
@@ -48,6 +48,9 @@
 
 #define TL_WA850RE_V2_GPIO_ENABLE_LEDS	15
 
+#define TL_WA855REV1_GPIO_LED_GREEN 12
+#define TL_WA855REV1_GPIO_LED_RED 505
```

Comment:
Something is wrong here, 505 isn't a correct GPIO number.
Also, please use tabs here (`#define XYZ[tab/s]N`).